worlds/shivers/__init__.py

```diff
@@ -65,153 +71,184 @@ def create_regions(self) -> None:
-                itempool.append(self.create_item(self.item_id_to_name[self.shivers_item_id_offset + 10 + i]))
-            elif self.options.full_pots == "complete":
-                itempool.append(self.create_item(self.item_id_to_name[self.shivers_item_id_offset + 20 + i]))
-            else:
-                # Roll for if pieces or a complete pot will be used.
-                # Pot Pieces
+        if self.options.full_pots == "pieces":
+            item_pool += [self.create_item(name) for name, data in item_table.items() if data.type == ItemType.POT]
+        elif self.options.full_pots == "complete":
+            item_pool += [self.create_item(name) for name, data in item_table.items() if
+                          data.type == ItemType.POT_COMPLETE]
+        else:
+            # Roll for if pieces or a complete pot will be used.
+            # Pot Pieces
+            pieces = [self.create_item(name) for name, data in item_table.items() if data.type == ItemType.POT]
+            complete = [self.create_item(name) for name, data in item_table.items() if
+                        data.type == ItemType.POT_COMPLETE]
+            for i in range(10):
                 if self.random.randint(0, 1) == 0:
                     self.pot_completed_list.append(0)
-                    itempool.append(self.create_item(self.item_id_to_name[self.shivers_item_id_offset + i]))
-                    itempool.append(self.create_item(self.item_id_to_name[self.shivers_item_id_offset + 10 + i]))
+                    item_pool.append(pieces[i])
+                    item_pool.append(pieces[i + 10])
                 # Completed Pot
                 else:
                     self.pot_completed_list.append(1)
-                    itempool.append(self.create_item(self.item_id_to_name[self.shivers_item_id_offset + 20 + i]))
+                    item_pool.append(complete[i])
 
-        #Add Filler
-        itempool += [self.create_item("Easier Lyre") for i in range(9)]
+        # Add Easier Lyre
+        item_pool += [self.create_item("Easier Lyre") for _ in range(9)]
 
-        #Extra filler is random between Heals and Easier Lyre. Heals weighted 95%.
-        filler_needed = len(self.multiworld.get_unfilled_locations(self.player)) - 24 - len(itempool)
-        itempool += [self.random.choices([self.create_item("Heal"), self.create_item("Easier Lyre")], weights=[95, 5])[0] for i in range(filler_needed)]
-
-        #Place library escape items. Choose a location to place the escape item
+        # Place library escape items. Choose a location to place the escape item
         library_region = self.multiworld.get_region("Library", self.player)
-        librarylocation = self.random.choice([loc for loc in library_region.locations if not loc.name.startswith("Accessible:")])
+        library_location = self.random.choice(
+            [loc for loc in library_region.locations if not loc.name.startswith("Storage:")]
```

Comment:
What you check for is inconsistent
```suggestion
            [loc for loc in library_region.locations if not loc.name.startswith("Storage: ")]
```